Automattic/wp-calypso
# Update action menu items in email plans page

#### Changes proposed in this Pull Request

This PR does the following:
* Makes the Google Admin console link visible to admin users only
* Adds the Edit item for email forwards
* Updates the titles for the Titan links to have "View" as a prefix
* Updates the hover styles for the icons in the action menu to be light (note the Google and Titan icons are not plain white, as that would lose some texture for those icons)

**Note:** This PR depends on #52912 for some general style fixes.

#### Testing instructions

* Run this branch locally or open the [live branch](https://calypso.live/?branch=update/email-plans-action-menu-items)
* Navigate to the email plan page for a domain with a Google Workspace or G Suite subscription where one of the mailboxes is not an admin.
* Verify that the menu for that mailbox does _not_ include the "View Admin" menu item.
* Hover over the menu items and verify that the icons are light and recognisable when you hover.
* Repeat the exercise for a Google Workspace/G Suite mailbox that is an admin, but verify that the "View Admin" menu item does appear.
* Navigate to the email plan page for an email forwarding account.
* Verify that a menu is present for each mailbox, and includes the "Edit" menu item.
* Hover over the "Edit" menu item, and verify that the icon is white on hover.
* Click on the "Edit" menu item, and verify that you're taken to the email forwarding management page.
* Navigate to the email plan page for a domain with a Professional Email subscription.
* Open the menu for a mailbox.
* Verify that the menu items all start with "View ".
* Hover over the menu items and verify that the icons are recognisable and much lighter.

#### Screenshots

##### Removed "View Admin" item for Google
<img width="750" alt="Screenshot 2021-05-18 at 09 29 56" src="https://user-images.githubusercontent.com/3376401/118610508-1ccf1480-b7bc-11eb-9eb9-1af93ad2c71b.png">

##### New "Edit" menu item for email forwards (with hover)
<img width="750" alt="Screenshot 2021-05-18 at 09 31 37" src="https://user-images.githubusercontent.com/3376401/118610506-1c367e00-b7bc-11eb-8d8c-f23aa37b98d8.png">

<img width="750" alt="Screenshot 2021-05-18 at 09 31 52" src="https://user-images.githubusercontent.com/3376401/118610501-1b055100-b7bc-11eb-82b3-c8b484bdbefb.png">

##### New display for existing icons on hover
<img width="494" alt="Screenshot 2021-05-18 at 09 27 43" src="https://user-images.githubusercontent.com/3376401/118610481-1771ca00-b7bc-11eb-81c8-42e372d8ec9e.png">

<img width="494" alt="Screenshot 2021-05-18 at 09 28 48" src="https://user-images.githubusercontent.com/3376401/118610514-1d67ab00-b7bc-11eb-9866-286a7b24f28a.png">


Related to #52912
